app/src/pages/feedbackPage/FeedbackPage.tsx

```diff
@@ -1,15 +1,166 @@
-import { SUBMISSION_STAGE } from '../../types/pages/feedbackPage/types';
 import React, { useState } from 'react';
-import FeedbackForm from '../../components/blocks/feedbackForm/FeedbackForm';
-import FeedbackConfirmation from '../../components/blocks/feedbackConfirmation/FeedbackConfirmation';
+import { SubmitHandler, useForm, UseFormRegisterReturn } from 'react-hook-form';
+import isEmail from 'validator/lib/isEmail';
+import { Button, Fieldset, Input, Radios, Textarea } from 'nhsuk-react-components';
+import { useNavigate } from 'react-router-dom';
+import { AxiosError } from 'axios';
+import {
+    FORM_FIELDS,
+    FormData,
+    SATISFACTION_CHOICES,
+    SUBMISSION_STAGE,
+} from '../../types/pages/feedbackPage/types';
+import SpinnerButton from '../../components/generic/spinnerButton/SpinnerButton';
+import useTitle from '../../helpers/hooks/useTitle';
+import useBaseAPIHeaders from '../../helpers/hooks/useBaseAPIHeaders';
+import useBaseAPIUrl from '../../helpers/hooks/useBaseAPIUrl';
+import sendEmail from '../../helpers/requests/sendEmail';
+import { isMock } from '../../helpers/utils/isLocal';
+import { routes } from '../../types/generic/routes';
+import { errorToParams } from '../../helpers/utils/errorToParams';
 
 function FeedbackPage() {
+    const baseUrl = useBaseAPIUrl();
+    const baseHeaders = useBaseAPIHeaders();
+    const {
+        handleSubmit,
+        register,
+        formState: { errors },
+    } = useForm<FormData>({
+        reValidateMode: 'onSubmit',
+    });
+    const navigate = useNavigate();
     const [stage, setStage] = useState(SUBMISSION_STAGE.NotSubmitted);
 
-    return stage === SUBMISSION_STAGE.Successful ? (
-        <FeedbackConfirmation />
-    ) : (
-        <FeedbackForm stage={stage} setStage={setStage} />
+    const submit: SubmitHandler<FormData> = async (formData) => {
+        setStage(SUBMISSION_STAGE.Submitting);
```

Comment:
It's being used for conditional rendering on line 151 so we do need some sort of state, do you mean change it to a boolean state instead of using the enum?